### PR TITLE
moved checkEditPermissionsAndUpdate function to higher scope

### PR DESCRIPTION
**Pull request by RebeccaStankus**
This is a fix for https://highfidelity.fogbugz.com/f/cases/14671/Error-in-edit-js-when-restarting-that-script

TEST PLAN

- Start Hifi dev build and open the log.
- Reload scripts (Ctrl + R) and search logs for "checkEditPermissionsAndUpdate". This line should not appear in the log at all.
- Open create menu and click on a few random items, then close create menu and check the log again. The line, "checkEditPermissionsAndUpdate", should still not appear in the log

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9474/app_13093_PR_7a7474bacaba00340d6c99add293d9e4c700c924.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9474/HighFidelity-Beta-PR13093-7a7474bacaba00340d6c99add293d9e4c700c924.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9474/HighFidelity-Beta-Interface-PR13093-7a7474bacaba00340d6c99add293d9e4c700c924.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9666/app_13093_PR_cb4d9586fefe754ee1b7923ffe888a2fcc423cef.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9666/HighFidelity-Beta-PR13093-cb4d9586fefe754ee1b7923ffe888a2fcc423cef.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9666/HighFidelity-Beta-Interface-PR13093-cb4d9586fefe754ee1b7923ffe888a2fcc423cef.dmg

**Comment by marko8904** [comment]
build this please

**Comment by conklin94122** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9677/HighFidelity-Beta-PR13093-cb4d9586fefe754ee1b7923ffe888a2fcc423cef.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9677/HighFidelity-Beta-Interface-PR13093-cb4d9586fefe754ee1b7923ffe888a2fcc423cef.dmg

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Line does not appear in the logs.

File: scripts/system/edit.js [hunk 1]
```diff
@@ -235,6 +235,24 @@ function adjustPositionPerBoundingBox(position, direction, registration, dimensi
 var TOOLS_PATH = Script.resolvePath("assets/images/tools/");
 var GRABBABLE_ENTITIES_MENU_CATEGORY = "Edit";
 
+// Handles any edit mode updates required when domains have switched
+function checkEditPermissionsAndUpdate() {
```
Comment (misslivirose) [inline scripts/system/edit.js]: Please also remove the other function - this is now defined twice in this file.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9920/HighFidelity-Beta-PR13093-93743538af11cd989d4207d883ca7b3fde704d7c.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9924/HighFidelity-Beta-PR13093-f4375d3e18c40dd291edd93ffcffc9aa0b5d13d8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9924/HighFidelity-Beta-PR13093-f4375d3e18c40dd291edd93ffcffc9aa0b5d13d8.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9924/HighFidelity-Beta-Interface-PR13093-f4375d3e18c40dd291edd93ffcffc9aa0b5d13d8.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9924/HighFidelity-Beta-PR13093-f4375d3e18c40dd291edd93ffcffc9aa0b5d13d8.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9924/HighFidelity-Beta-Interface-PR13093-f4375d3e18c40dd291edd93ffcffc9aa0b5d13d8.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 
Line does not appear on logs

**Pull request merged**